torch/csrc/jit/passes/quantization.cpp

```diff
@@ -106,22 +106,24 @@ std::string getFuncName(Value* func_value) {
   }
 }
 
-bool nodeQuantizable(Node* n) {
-  static std::vector<std::string> call_funcs = {
-      "conv2d",
-      "linear",
-      "relu",
-  };
-  std::vector<Symbol> aten_funcs = {
-      Symbol::aten("addmm"), Symbol::aten("matmul"), Symbol::aten("add_")};
+bool isFunctionNode(Node* n,
+                    const std::vector<std::string>& call_funcs,
+                    const std::vector<std::string>& aten_funcs) {
+  std::vector<Symbol> all_funcs;
   std::transform(
```

Comment:
Could this be written as

```
for (const auto& funcs : {call_funcs, aten_funcs}) {
  for (const auto& s : funcs) {
    all_funcs.push_back(s);
  }
}
```

Reply:
I mean `push_back(Symbol::aten(s))`

Reply:
changed the logic a little bit, I'm going to list out both call functions and aten functions separately.